cms/static/js/spec/views/active_video_upload_list_spec.js

```diff
@@ -211,6 +224,18 @@ define(
                                                 expect($uploadElem).not.toHaveClass("queued");
                                             });
                                         }
+
+                                        // If we're uploading more files than the one we've closed above, 
+                                        // the unload warning should still be shown
+                                        if (caseInfo.numFiles > 1) {
+                                            it('should show a notification message when videos are still uploading', function () {
+                                                expect(this.view.onBeforeUnload()).toBe(this.view.pageUnloadMessage);
```

Comment:
This test is fine, but it doesn't actually test that the notification message is what you expect it to be. If some other function modified the `view.pageUnloadMessage` to be "HAHA YOU'RE GONNA LOSE ALL YOUR DATA", this test would still pass just fine. You may want to actually assert on what you expect the message to *be*, rather than asserting that it matches the content of another variable.